src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,8 +1,61 @@
 package core.basesyntax;
 
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.util.Collections;
+import java.util.Comparator;
+import java.util.LinkedHashMap;
+import java.util.List;
+import java.util.Map;
+
 public class WorkWithFile {
 
     public void getStatistic(String fromFileName, String toFileName) {
+        Map<String, Integer> hashMap = new LinkedHashMap<>();
```

Comment:
can we use HashMap here?

Reply:
Make without using Hash Map?

Reply:
I meant, you could use HashMap instead of LinkedHashMap